bounter/count_min_sketch.py

```diff
@@ -181,3 +182,11 @@ def __getstate__(self):
     def __setstate__(self, state):
         self.width, self.depth, self.cell_size_v, self.cms = state
         self.increment = self.cms.increment
+
+
+class CardinalityEstimator(CountMinSketch):
+    def __init__(self):
+        super(CardinalityEstimator, self).__init__(width=1, depth=1)
+
+    def __getitem__(self, key):
+        raise NotImplementedError("Item counting is not supported for cardinality estimator!")
```

Comment:
`Individual item counting`, to prevent misunderstandings. Also, no need for the `!` IMO (just my style preference :).